runtime/manta/Cargo.toml

```diff
@@ -43,7 +45,6 @@ pallet-authorship = { git = 'https://github.com/paritytech/substrate.git', defau
 pallet-balances = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.26" }
 pallet-multisig = { git = 'https://github.com/paritytech/substrate.git', default-features = false, branch = "polkadot-v0.9.26" }
```

Comment:
Seems ~~`pallet-multisig`~~, `pallet-utility` and `pallet-preimage` can be removed. But not 100% sure.

Reply:
won't we need `pallet-preimage` to do runtime upgrades through the `authorize_upgrade` => `enact_authorized_upgrade` flow?

For utility reasoning see https://github.com/Manta-Network/Manta/pull/946#discussion_r1057814981